cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```diff
@@ -276,6 +284,26 @@ objects:
       usePodIdentity: "false"
     provider: azure
 
+- apiVersion: secrets-store.csi.x-k8s.io/v1
+  kind: SecretProviderClass
+  metadata:
+    name: arm-helper-identity
+    namespace: ${NAMESPACE}
+  spec:
+    parameters:
+      clientID: ${AZURE_CS_MI_CLIENT_ID}
+      cloudName: AzurePublicCloud
+      keyvaultName: ${SERVICE_KEYVAULT_NAME}
+      objects: |-
+        array:
+          - |
+            objectName: ${AZURE_ARM_HELPER_IDENTITY_CERT_NAME}
+            objectType: secret
+            objectAlias: armHelperIndentityCertificateBundle
+      tenantId: ${TENANT_ID}
+      usePodIdentity: "false"
+    provider: azure
```

Comment:
nit; I think we can re-use the `cs-keyvault`  SecretProviderClass and add the 
```
        objectName: ${AZURE_ARM_HELPER_IDENTITY_CERT_NAME}
            objectType: secret
            objectAlias: armHelperIndentityCertificateBundle
```

in its list of objects

Reply:
removed